resources/resource_ids.spec

```diff
@@ -130,13 +130,9 @@
   },
   # This file is generated during the build.
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-nft_display/nft_display.grd": {
-    "META": {"sizes": {"includes": [10]}},
+    "META": {"sizes": {"includes": [20]}},
     "includes": [34070],
   },
-  "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-cookie_list_opt_in/cookie_list_opt_in.grd": {
-    "META": {"sizes": {"includes": [10]}},
-    "includes": [34080],
-  },
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-brave_speedreader_toolbar/brave_speedreader_toolbar.grd": {
```

Comment:
@mkarolin is it just a manual process to bump all of these back? Should we do it now or wait until we need more?

Reply:
Yes, it's a manual process. Probably best to update with the removal.

Reply:
Let's just do your removal change and then I'll create a follow-up issue to use the tool to renumber all of them.